packages/backend/src/models/UserAttributesModel.ts

```diff
@@ -212,6 +253,26 @@ export class UserAttributesModel {
         await Promise.all(promises);
     }
 
+    private static async insertGroupAttributes(
+        trx: Knex.Transaction,
+        userAttributeUuid: string,
+        groups: { groupUuid: string; value: string }[],
+    ): Promise<void> {
+        const promises = groups.map(async (groupAttr) => {
+            const [group] = await trx(`groups`)
+                .where(`group_uuid`, groupAttr.groupUuid)
+                .select('group_uuid');
```

Comment:
I don't think we need this query. 
When inserting in `GroupUserAttributesTable` the group_uuid column is a foreign key, so if the group_uuid doesn't exist in that table it will throw an error.

Reply:
Good point.